client/finality-grandpa/src/communication/gossip.rs

```diff
@@ -562,27 +562,43 @@ impl<N: Ord> Peers<N> {
 		self.inner.get(who)
 	}
 
-	fn authorities(&self) -> usize {
-		self.inner.iter().filter(|(_, info)| matches!(info.roles, ObservedRole::Authority)).count()
+	fn connected_authorities(&self) -> usize {
+		self.inner
+			.iter()
+			.filter(|(_, info)| matches!(info.roles, ObservedRole::Authority))
+			.count()
 	}
 
-	fn non_authorities(&self) -> usize {
+	fn connected_full(&self) -> usize {
```

Comment:
nitpick: consider maybe call this `connected_full_nodes`
(even though it would be asymmetric wrt the enum...)